### PR TITLE
[chore] Update `update_resource` schema

### PR DESCRIPTION
**Pull request by kgrhartlage**
### Description

The relevant changes are already in production, we missed opening a corresponding PR in this repo.

I also updated the descriptions and added examples.

> **Note** I want to rename the file before merging.

### CODE REVIEW

File: src/schemata/definitions/workflow_template/action_objects/updateResource.yml [hunk 1]
```diff
@@ -33,12 +33,55 @@ additionalProperties:
...
+                  $ref: '#/definitions/script'
+                  description: |
+                    To update a custom attribute, the script must evaluate to a hash object with `name`, `type` and `value` attributes. The supported custom attribute data types are `number`, `text`, `boolean` (checkbox), `date`, `datetime`, `file`, `image`, `physicalQuantity`, `relation`, and `duration`.
 examples:
-  - update_resource:
-      id: workflow_run.data.id
-      type: workflow_run
-      attributes:
-        title: sampleField.name
+  - id: workflow_run.data.id
```
Comment (ignatiusreza) [inline src/schemata/definitions/workflow_template/action_objects/updateResource.yml:68]: why did you remove the `update_resource:` key from the examples?

File: src/schemata/definitions/workflow_template/action_objects/updateResource.yml [hunk 2]
```diff
@@ -33,12 +33,55 @@ additionalProperties:
               type: object
               additionalProperties: false
               properties:
+                custom_attributes:
+                  $ref: '#/definitions/script'
+                  description: |
+                    To update a custom attribute, the script must resolve to a hash object with `name`, `type` and `value` attributes. The supported custom attribute data types are `number`, `text`, `boolean` (checkbox), `date`, `datetime`, `file`, `image`, `physicalQuantity`, `relation`, and `duration`.
```
Comment (ignatiusreza) [inline src/schemata/definitions/workflow_template/action_objects/updateResource.yml:39]: only `name` and `value` is needed.. I believe.. :thinking: 

type is being defined by the custom attributes definition in the system administration page..

**Review by ignatiusreza — CHANGES_REQUESTED** [review]

File: src/schemata/definitions/workflow_template/action_objects/updateResource.yml (hunk 1 above)
Comment (kgrhartlage) [reply src/schemata/definitions/workflow_template/action_objects/updateResource.yml:68]: I added the key

File: src/schemata/definitions/workflow_template/action_objects/updateResource.yml (hunk 2 above)
Comment (kgrhartlage) [reply src/schemata/definitions/workflow_template/action_objects/updateResource.yml:39]: Nope, does not seem to work. Tested with this template

<details>
<summary>template</summary>

```yml
---
schema_version: '0.0.1'
info:
  version: '55.0.0'
  uuid: '8fc60467-f93a-4d2c-b6f3-84ec6e053496'
  title: 'Update Resource'
fields: {}
steps:
  stepOne:
    info:
      title: 'Show Alert'
    substeps:
      - primary: 'Now, look at this field!'
        expandDescription: true
        description: |
          Update all the things!
        buttons:
          - label: Click me
            key: alert-button

        behaviors:
          - when:
              manual:
                key: alert-button
            do:
              - update_resource:
                  id: 5
                  type: collection
                  attributes:
                    custom_attributes: |
                      {
                        HASH(
                          "name","myText1",
                          "type", "text",
                          "value", "with type"
                        ),
                        HASH(
                          "name","myText2",
                          "value", "no type"
                        )
                      }

flow:
  - stepOne

```

</details>

**Comment by kgrhartlage** [comment]
I renamed the file. The naming convention is all over the place and we decided to go with `file-name.yml`. I will update the rest all together in a dedicated PR soon-ish

File: src/schemata/definitions/workflow_template/action_objects/updateResource.yml (hunk 2 above)
Comment (ignatiusreza) [reply src/schemata/definitions/workflow_template/action_objects/updateResource.yml]: yeah, that's because the custom attribute is not yet defined in the system administration page..

we can keep the changes in this PR as is for now, and we can revisit this again after we fully remove the support for local custom attribute..

**Review by ignatiusreza — APPROVED** [review]
